templates/documentation/runbooks/pod-memory.md.tpl

```diff
@@ -10,12 +10,60 @@
 
 # {{ camelcase .Config.Name }} Pod Memory.\<type\> > 80% of limit last 30m
 
+Memory high alerts will occur if the memory rate is too high over x amount of time. The net impact is that customers might start seeing failures in {{ camelcase .Config.Name }} requests. On the other hand, none of the above might actually happen and the memory usage might drop down on it’s own. If the High Memory usage keeps repeating for the same host, there might be something wrong with the host itself and it might need recycling - all of this is described in much more detail below.
+
+## Investigation
+
+### Datadog Dashboard and Logs
+
 [Navigate to the `Terraform: {{ camelcase .Config.Name }}` dashboard]({{ $dashboard }}) and find the "Total Memory" monitor under
 the "Deployment" pane. Zero-in on a time frame where memory spiked, note that time frame, and change the window of
 the dashboard to that time frame. Correlate any other useful monitors to see what could be causing this - look at
 various sources of traffic like gRPC or HTTP. Looking at logs at the same time frame may also be useful using the
 `kube_namespace:{{ .Config.Name }}--<bento>` facet.
 
+Also look at the [trending memory dashboard](https://app.datadoghq.com/dashboard/i6z-jcn-azh/terraform-{{ .Config.Name }}?fullscreen_end_ts=1642528770105&fullscreen_paused=false&fullscreen_section=overview&fullscreen_start_ts=1641923970105&fullscreen_widget=5625648833266685&from_ts=1641923966020&to_ts=1642528766020&live=true) in DataDog.
```

Comment:
issue: this link will not work, it's specific to usermanager. It's better to describe how to get to the trending memory section of the dashboard.